src/test/datascience/notebook/executionService.vscode.test.ts

```diff
@@ -123,6 +123,14 @@ suite('DataScience - VSCode Notebook - (Execution) (slow)', function () {
 
         await Promise.all([runCell(cell), waitForTextOutput(cell, '123412341234')]);
     });
+    test('Test __vsc_ipynb_file__ defined in cell using VSCode Kernel', async () => {
+        const uri = vscodeNotebook.activeNotebookEditor?.document.uri;
+        if (uri && uri.scheme === 'file') {
+            await insertCodeCell('print("__vsc_ipynb_file__")', { index: 0 });
```

Comment:
Actually no, I'm wrong. It fails on all tests. This isn't a unit test, it's an execution test. 

The code should be this:

```
await insertCodeCell('print(__vsc_ipynb_file__)');
```